apps/web/lib/d/[link]/[slug]/getServerSideProps.tsx

```diff
@@ -72,6 +117,31 @@ async function getUserPageProps(context: GetServerSidePropsContext) {
   if (!hashedLink) {
     return notFound;
   }
+
+  const isExpired = hashedLink.expiresAt
+    ? (() => {
+        const hostTimezone = HashedLinksService.extractHostTimezone(hashedLink.eventType);
+
+        if (hostTimezone) {
+          const now = dayjs().tz(hostTimezone);
+          const expiration = dayjs(hashedLink.expiresAt).tz(hostTimezone);
+          return expiration.isBefore(now);
+        } else {
+          const now = dayjs();
+          const expiration = dayjs(hashedLink.expiresAt);
+          return expiration.isBefore(now);
+        }
+      })()
+    : false;
+  const isUsageExceeded = hashedLink.maxUsageCount
```

Comment:
We don't reset `maxUsageCount` for time-based expiration. Wouldn't that cause `isUsageExceeded` always be true for time-based expiration?